### DISCUSSION TITLE
Santoker R3 USB Connection // Which driver?

### DISCUSSION THREAD

**Discussion by mattimadaro** [artisan-roaster-scope/artisan#1582] [Q&A]
Hi all! 
I want to connect Artisan to the Santoker R3 via Bluetooth/WIFI or USB connection. Does Artisan already support Bluetooth connection on this specific roaster for Win and Mac? If not, which specific driver (for USB connection) do I need for this roaster? Any help is much appreciated as I could not find an answer elsewhere.

Mattia

**Comment by poundy** [comment]
Here is the supported machines page entry for Santoker. https://artisan-scope.org/machines/santoker/
According to that, the R3 is a USB connection OR a wifi connection. The page has a link to a serial driver for USB connection.  Did you go through the machine setup process ? Do you know what the IP address the machine has been allocated on the wifi network?

**Reply by mattimadaro** [reply]
Thanks for replying. I have just plugged in the USB cable, but did not install the driver. I will try to set this up and return if I have more questions. My friend roasting on the same machine could not connect his Mac. Maybe it's the same issue. Thanks again.

**Comment by Terracotta-6** [comment]
Not sure it's right. USB cable
Driver for the USB serial converter:
https://www.silabs.com/documents/public/software/Mac_OSX_VCP_Driver.zip
https://www.silabs.com/documents/public/software/CP210x_Universal_Windows_Driver.zip

SANTOKER Coffee Tech

[Hunan Santoker Intelligent Technology Co., Ltd. (alibaba.com)](https://santoker.en.alibaba.com/?spm=a2700.galleryofferlist.wending_right.4.7ecf2c52uGNFSB)

 

Address: 40-2, gaobeidian xincun, chaoyang district, Beijing 

Tel: + 86 15116517222

Website: www.santoker.com 

Email address: hustnan@santoker.com

Wechat：santoker

 

 

Distributor Thailand：

SANTOKER(THAILAND) CO.,LTD.

 

Contact：Tan

Phone number：065-0544493

Email：santoker.thailand@gmail.com

Address：11/3 M.6 Saimoon，Sankhumpheng，Chiangmai, Thailand

 

HENAN WINTOP MACHINERY TECHNOLOGY CO., LTD
Contactor：Jenny Tse 谢豫彬
MP：18203850626
Email：info@wintopmachinery.com

**Reply by poundy** [reply]
the page linked to gives a number of possible drivers. The person doing this would need to identify what driver was missing (on Windows, use Device Manager) and select the right one...
